telemetry/definitions/commonDefinitions.json

```diff
@@ -773,6 +773,61 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "profileNameStatus",
+            "type": "string",
+            "description": "Shows the status of Profile Name input validations under IAM Credentials",
+            "allowedValues":[
+                "empty", "valid", "invalid"
+            ]
+        },
+        {
+            "name": "accessKeyStatus",
+            "type": "string",
+            "description": "Shows the status of AccessKey input validations under IAM Credentials",
+            "allowedValues":[
+                "empty", "valid", "invalid"
+            ]
+        },
+        {
+            "name": "secretStatus",
+            "type": "string",
+            "description": "Shows the status of Secret Key input validations under IAM Credentials",
+            "allowedValues":[
+                "empty", "valid", "invalid"
+            ]
+        },
+        {
+            "name": "numberOfConnections",
+            "type": "int",
+            "description": "Number of Existing connections the user has."
+            
+        },
+        {
+            "name": "builderIDStatus",
+            "type": "string",
+            "description": "If the user if successfully logged in",
+            "allowedValues":[ "connected", "non-connected"
+            ]
+            
```

Comment:
TODO: add auto-formatting for JSON files to the `.vscode` settings

Reply:
any reason this has to be string instead of a boolean?